torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -189,13 +243,19 @@ def __init__(
         self.transform = transform
         self.neighbor_sampler = neighbor_sampler
 
+        batch_size = kwargs.get("batch_size", 1)
```

Comment:
I was thinking of instead just making negative samples make the batch size larger.. but this seemed like the lesser evil to me